README.md

```diff
@@ -33,13 +33,13 @@ self.view.addSubview(mapView)
 
 mapView.rx.didChangePosition.asDriver()
     .drive(onNext: { print("Did change camera position: \($0)") })
-    .addDisposableTo(disposeBag)
+    .disposed(by: disposeBag)
```

Comment:
Except for the syntactical fixes here, are there any updates to the examples themselves that stem from your massive refactor here? Or the code should be relatively similar? Perhaps an update to the README is in order?